src/get-started/test-service.md

```diff
@@ -59,22 +59,16 @@ contain the tests for transaction business logic.
 
 ### Imports
 
-The created file is executed separately from the service code, meaning that we
-need to import the service crate along with **exonum** and **exonum-testkit**:
-
-```rust
-extern crate exonum;
-extern crate exonum_cryptocurrency as cryptocurrency;
-#[macro_use] extern crate exonum_testkit;
-```
-
 Just like with the service itself, we then import the types we will use:
 
 ```rust
-use exonum::blockchain::Transaction;
-use exonum::crypto::{self, PublicKey, SecretKey};
-use exonum_testkit::{TestKit, TestKitBuilder};
-// Import datatypes used in tests from the crate where the service is defined.
+use assert_matches::assert_matches;
+use exonum::api::Error;
+use exonum::messages::{to_hex_string, RawTransaction, Signed};
+use exonum_crypto::{gen_keypair, PublicKey, SecretKey};
+use exonum_testkit::{txvec, ApiKind, TestKit, TestKitApi, TestKitBuilder};
+use serde_json::json;
+// Imports datatypes used in tests from the crate where the service is defined.
 use cryptocurrency::schema::{CurrencySchema, Wallet};
```

Comment:
Shouldn't the crate name be changed to `exonum_cryptocurrency`? I'd also group the `use` statements here as well.

Reply:
I'm not sure about renaming to `exonum_cryptocurrency`. As I understand, in this tutorial we create integration tests for just created cryptocurrency example, don't we ?

Reply:
But let it be. Not principle.